<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>WinInfo</class>
    <widget class="QDialog" name="WinInfo">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>790</width>
                <height>565</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Information</string>
        </property>
        <layout class="QFormLayout" name="layout">
            <item row="0" column="0">
                <widget class="QLabel" name="labelName">
                    <property name="text">
                        <string>Title</string>
                    </property>
                </widget>
            </item>
            <item row="0" column="1">
                <widget class="QLabel" name="txtName">
                    <property name="text">
                        <string/>
                    </property>
                    <property name="wordWrap">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
            <item row="1" column="0">
                <widget class="QLabel" name="labelBVID">
                    <property name="text">
                        <string>BVID</string>
                    </property>
                </widget>
            </item>
            <item row="1" column="1">
                <layout class="QHBoxLayout" name="layoutBVID">
                    <item>
                        <widget class="QLabel" name="txtBVID">
                            <property name="text">
                                <string/>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="btnBVID">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="text">
                                <string>...</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item row="2" column="0">
                <widget class="QLabel" name="labelAuth">
                    <property name="text">
                        <string>Author</string>
                    </property>
                </widget>
            </item>
            <item row="2" column="1">
                <layout class="QHBoxLayout" name="layoutAuth">
                    <item>
                        <widget class="QLabel" name="txtAuth">
                            <property name="text">
                                <string/>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="btnAuth">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="text">
                                <string>...</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item row="3" column="0">
                <widget class="QLabel" name="labelDesc">
                    <property name="text">
                        <string>Description</string>
                    </property>
                </widget>
            </item>
            <item row="3" column="1">
                <widget class="QTextBrowser" name="txtDesc"/>
            </item>
            <item row="4" column="0" colspan="2">
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Close</set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>WinInfo</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>248</x>
                    <y>254</y>
                </hint>
                <hint type="destinationlabel">
                    <x>157</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>WinInfo</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>316</x>
                    <y>260</y>
                </hint>
                <hint type="destinationlabel">
                    <x>286</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
